pom.xml

```diff
@@ -2332,6 +2332,20 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>fast</id>
+            <properties>
+                <checkstyle.skip>true</checkstyle.skip>
+                <maven.javadoc.skip>true</maven.javadoc.skip>
+                <maven.resources.skip>true</maven.resources.skip>
```

Comment:
it disables the copy jars? it's not expensive, i'd like to keep it.

Reply:
OK, removed.